subprojects/scala/src/main/java/org/gradle/language/scala/tasks/BaseScalaCompileOptions.java

```diff
@@ -225,4 +230,26 @@ public IncrementalCompileOptions getIncrementalOptions() {
     public void setIncrementalOptions(IncrementalCompileOptions incrementalOptions) {
         this.incrementalOptions = incrementalOptions;
     }
+
+    /**
+     * Keeps Scala compiler daemon alive across builds for faster build times
+     * Legal values:
+     * - SESSION (compiler is kept alive for a session - default)
+     * - DAEMON (compiler is kept alive across builds )
```

Comment:
Please run the `sanityCheck` task on the project.
The property might need a `@since 7.6` javadoc annotation and an `@Incubating` java annotation.